packages/sdk-components-react-radix/src/radix-collapsible.ws.ts

```diff
@@ -0,0 +1,106 @@
+import { RadioUncheckedIcon, RadioCheckedIcon } from "@webstudio-is/icons/svg";
+import type {
+  WsComponentMeta,
+  WsComponentPropsMeta,
+} from "@webstudio-is/react-sdk";
+import {
+  propsRadixCollapsible,
+  propsRadixCollapsibleContent,
+  propsRadixCollapsibleTrigger,
+} from "./__generated__/radix-collapsible.props";
+
+export const metaRadixCollapsible: WsComponentMeta = {
+  category: "radix",
+  type: "container",
+  label: "Collapsible",
+  icon: RadioUncheckedIcon,
+  template: [
```

Comment:
Styles will be added later?

Reply:
There is no styles in shaden for collapsible. Purely functional component.

Reply:
shadcn has tyles there, no?
https://ui.shadcn.com/docs/components/collapsible

Reply:
Those are example styles